include/bluetooth/mesh/cfg_cli.h

```diff
@@ -56,6 +56,10 @@ int bt_mesh_cfg_node_reset(uint16_t net_idx, uint16_t addr, bool *status);
  *  return the largest page number it supports that is less than the requested
  *  page index. The actual page the device responds with is returned in @c rsp.
  *
+ *  This method can be used asynchronously by setting @p rsp or @p comp
+ *  as NULL. This way the method will not wait for response and will return
+ *  immediately after sending the command.
```

Comment:
For these two-parameter response functions; I think it would be more sensible to make the function async only if both parameters are NULL. You might want to see that status, but not the resulting phase, for instance

Reply:
Sounds reasonable.